nncf/experimental/torch/fx/transformations.py

```diff
@@ -0,0 +1,398 @@
+            else:
+                # for qparams that are not scale/zero_point (like axis, dtype) we store
+                # them as literals in the graph.
+                quantize_op_inputs.append(value_or_node)
+    with graph.inserting_after(target_node):
+        quantized_node = graph.create_node(node_type, quantize_op, tuple(quantize_op_inputs), {})
+        # use the same qparams from quantize op
+    dq_inputs = [quantized_node] + quantize_op_inputs[1:]
+    user_dq_nodes = []
+    with graph.inserting_after(quantized_node):
+        for user in target_node.users:
+            if user is quantized_node:
+                continue
+            user_dq_nodes.append((user, graph.call_function(dequantize_op, tuple(dq_inputs), {})))
+
+    for user, dq_node in user_dq_nodes:
+        user.replace_input_with(target_node, dq_node)
+
+
+def _insert_call_module(graph: torch.fx.Graph, target_node: torch.fx.Node, module_attr_name: str):
+    """
+    Inserts module call node to the graph after the target node.
+
+    :param graph: Graph to insert module call node.
+    :param target_node: Target node, module call node is being iserted just after the target node.
+    :param module_attr_name: The name of the graph attribute which keeps the target module.
+    """
+    with graph.inserting_after(target_node):
+        return graph.create_node(
+            "call_module", module_attr_name, (target_node,), {}, name=module_attr_name + "_graph_node"
+        )
+
+
+def _get_target_node(graph: torch.fx.Graph, target_point: PTTargetPoint) -> torch.fx.Node:
+    """
+    Returns TorchFX graph node correspondent to the target point.
+
+    :param graph: Target torch.fx.Graph.
+    :param target_point: A target point to find the target node.
+    :return: TorchFX graph node correspondent to the target point.
+    """
+    target_type = target_point.target_type
+    target_node = FXModelTransformer.get_graph_node_by_name(graph, target_point.target_node_name)
+    if target_type in [TargetType.OPERATOR_PRE_HOOK, TargetType.OPERATION_WITH_WEIGHTS]:
+        target_node = target_node.all_input_nodes[target_point.input_port_id]
+    elif target_type == TargetType.OPERATOR_POST_HOOK:
+        pass
+    else:
+        raise RuntimeError(f"Unsupported target type: {target_type} for target_point: {target_point}")
+    return target_node
+
+
+def _set_module_to_the_graph_module(
+    model: torch.fx.GraphModule, module_to_insert: torch.nn.Module, target_points: List[PTTargetPoint]
+) -> str:
+    """
+    Sets given module to the given torch.fx.GraphModule with unique name.
+
+    :param graph: Target torch.fx.Graph.
+    :param module_to_insert: Module to insert to the target graph.
+    :param target_points: Target points which will be used to insert target module
+        to the graph.
+    :return: A graph module attribute name which keep given module.
+    """
+    module_to_insert = module_to_insert
+    module_name_in_model = (
+        ";".join(
```

Comment:
';' symbol is used in `create_shared_quantizer_insertion_command` function, so I reused it https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/graph/transformations/command_creation.py#L75

What symbol do you recommend instead? I'm afraid there could be conflicts in naming in case we omit any attribute from the name

Reply:
Could you clarify `storage_key` from https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/graph/transformations/command_creation.py#L75 is a dict key?

Reply:
It is used [here](https://github.com/openvinotoolkit/nncf/blob/develop/nncf/torch/nncf_network.py#L588-L597) in the `add_compression_module` method as an `module_key`

Reply:
I suggest to replace ";" to "__"

Reply:
Agree, done